docs/release-notes/cloud/cloud-2023-07.md

```diff
@@ -0,0 +1,167 @@
+-   https://codacy.atlassian.net/browse/IO-550
+-   https://codacy.atlassian.net/browse/HRZ-282
+-   https://codacy.atlassian.net/browse/HRZ-281
+-   https://codacy.atlassian.net/browse/HRZ-265
+-   https://codacy.atlassian.net/browse/HRZ-126
+-   https://codacy.atlassian.net/browse/HRZ-86
+-   https://codacy.atlassian.net/browse/CY-6865
+-   https://codacy.atlassian.net/browse/CY-6534
+-   https://codacy.atlassian.net/browse/CY-4844
+-   https://codacy.atlassian.net/browse/CY-4798
+-   https://codacy.atlassian.net/browse/ALA-464
+-   https://codacy.atlassian.net/browse/ALA-460
+-   https://codacy.atlassian.net/browse/ALA-451
+-   https://codacy.atlassian.net/browse/ALA-282
+-   https://codacy.atlassian.net/browse/ALA-151
+-   https://codacy.atlassian.net/browse/ALA-110
+Bugs and other issues:
+-   https://codacy.atlassian.net/browse/TS-500
+-   https://codacy.atlassian.net/browse/TS-496
+-   https://codacy.atlassian.net/browse/TS-494
+-   https://codacy.atlassian.net/browse/TS-488
+-   https://codacy.atlassian.net/browse/TS-483
+-   https://codacy.atlassian.net/browse/TS-482
+-   https://codacy.atlassian.net/browse/TS-481
+-   https://codacy.atlassian.net/browse/TS-480
+-   https://codacy.atlassian.net/browse/TS-479
+-   https://codacy.atlassian.net/browse/TS-478
+-   https://codacy.atlassian.net/browse/TS-475
+-   https://codacy.atlassian.net/browse/TS-474
+-   https://codacy.atlassian.net/browse/TS-472
+-   https://codacy.atlassian.net/browse/TS-471
+-   https://codacy.atlassian.net/browse/TS-469
+-   https://codacy.atlassian.net/browse/TS-465
+-   https://codacy.atlassian.net/browse/TS-461
+-   https://codacy.atlassian.net/browse/TS-460
+-   https://codacy.atlassian.net/browse/TS-456
+-   https://codacy.atlassian.net/browse/TS-451
+-   https://codacy.atlassian.net/browse/TS-450
+-   https://codacy.atlassian.net/browse/TS-436
+-   https://codacy.atlassian.net/browse/TS-422
+-   https://codacy.atlassian.net/browse/TS-399
+-   https://codacy.atlassian.net/browse/TS-336
+-   https://codacy.atlassian.net/browse/TS-313
+-   https://codacy.atlassian.net/browse/PUL-2201
+-   https://codacy.atlassian.net/browse/PUL-2186
+-   https://codacy.atlassian.net/browse/PUL-2182
+-   https://codacy.atlassian.net/browse/PUL-2174
+-   https://codacy.atlassian.net/browse/PUL-2172
+-   https://codacy.atlassian.net/browse/IO-731
+-   https://codacy.atlassian.net/browse/IO-664
+-   https://codacy.atlassian.net/browse/HRZ-304
+-   https://codacy.atlassian.net/browse/HRZ-300
+-   https://codacy.atlassian.net/browse/HRZ-299
+-   https://codacy.atlassian.net/browse/HRZ-298
+-   https://codacy.atlassian.net/browse/HRZ-291
+-   https://codacy.atlassian.net/browse/ALA-490
+-   https://codacy.atlassian.net/browse/ALA-489
+-   https://codacy.atlassian.net/browse/ALA-482
+-->
+
+## Product enhancements
+
+-   You can now [use Security and risk management to monitor Jira security issues](../../organizations/managing-security-and-risk.md#managing-integrations). (PUL-1997)
```

Comment:
@mrfyda Should it mention what is, what for a Security Manager?

Reply:
We [mention Security Managers](https://github.com/codacy/docs/blob/b92d610001ad3c6410004ead0cfcad9c6679752d/docs/release-notes/cloud/cloud-2023-07.md?plain=1#L22) it in the updated version 👍